pointercrate-demonlist-pages/static/css/statsviewer.css

```
@@ -49,7 +49,11 @@
     flex-grow: 2;
 }
 
+.name-main-span:hover {
+    cursor: pointer;
+    text-decoration: underline;
```

```suggestion
    border-bottom: 1px dotted #555;
```

that's how these "hover for more" texts are styled elsewhere on pointercrate (it's crammed in the tooltip CSS class that's used for the `more` tooltips when there's more than 3 creators. Maybe this is a good opportunity to generalize that, so that we don't repeat the style rule here?)